examples/login-data-example/login-data-example.html

```diff
@@ -4,7 +4,8 @@
   <meta charset="UTF-8">
   <title>VGS Collect Login data Example</title>
 
-  <!--Include generated js file to your application-->
+  <!--Include generated for your organization JS file into an application.
+      File below is just our example. Please use your own JS file inside your application.-->
   <script src="https://js.verygoodvault.com/js-vgfield-2/development.js"></script>
```

Comment:
what do you think about to remove a real file and include something like this?
`<script src="path/to/your/personal/VGS-Collect/library/filename.js"></script>`

It will push users to generate/download their own library and try to run the example with his own vault.